packages/@aws-cdk/aws-cognito/README.md

```diff
@@ -741,6 +741,19 @@ pool.addClient('app-client', {
 });
 ```
 
+User Pool clients can generate a client ID as well as a client secret, to support more advanced authentication workflows.
+
+To create a client with an autogenerated client secret, pass the `generateSecret: true` prop:
+
+```ts
+const userPoolClient = new cognito.UserPoolClient(this, 'UserPoolClient', {
+  userPool: importedPool,
+  generateSecret: true,
+});
+
+// Allows you to pass the generated secret to other pieces of infrastructure
+const secret = userPoolClient.userPoolClientSecret;
+```
```

Comment:
Add an extra newline here - it will probably fix the broken build.

Reply:
PR linter now passes green.